src/common/icons/Icons.tsx

```diff
@@ -373,7 +379,7 @@ export const CheckedSquareIcon = (): JSX.Element => (
 )
 
 export const ExclamationTriangleIcon = (): JSX.Element => (
-  <IconContainer suppressIconStyles className="fa fa-exclamation-triangle" />
+  <IconContainer className="fa fa-exclamation-triangle" />
```

Comment:
This `suppressIconStyles` you checked didn't do anything if I remember correctly?

Reply:
yes, it is never used